media/js/firefox/sync.js

```diff
@@ -126,6 +126,12 @@
         'data-download-version': 'Firefox for Android'
     });
 
+    // Firefox Sync sign in flow button
+    $('#cta-sync').on('click', function(e) {
+        e.preventDefault();
+        Mozilla.UITour.showFirefoxAccounts();
```

Comment:
Yep - this is not possible with Casper since it requires a real Firefox with custom preferences. Need to work out something for the future - perhaps a custom build with Selenium